client/src/components/photoupload.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from "react";
+import axios from "axios";
+import "../App.css";
+
+class RenderProfilePage extends Component {
```

Comment:
I haven't yet pushed my changes, but I've made the routes, and models for the picture uploading. I was not able to get pictures using the previous changes made for picture uploads, but this time I got a picture to display on the screen. I believe it can be integrated with this photoupload component.